packages/components/src/theme.js

```diff
@@ -1,5 +1,11 @@
 module.exports = {
   theme: {
+    backgroundSize: {
+      "100-200": "100% 220%",
+    },
+    backgroundPosition: {
+      bottom: "0% 99%", // flickering on firefox
```

Comment:
Hmm, I'm not sure if we should override the bottom position with `0 99%`. This seems to lead to other problems in the future.